lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -541,6 +542,7 @@ def cleanup_provider_classes():
 
     @patch('ccx.views.render_to_response', intercept_renderer)
     def test_gradebook(self):
+        self.course.enable_ccx = True
```

Comment:
@cguardia Do we need to check enable_ccx = False, what if user edits grading policy when ccx enable, can we access new grading policy.
2nd can we test the custom and original grading policy is different?